share/smack/top.py

```diff
@@ -367,6 +369,39 @@ def objc_clang_frontend(args):
   compile_command = objc_clang_compile_command(args)
   default_link_bc_files(compile_command, args)
 
+def fortran_frontend(args):
+  """Generate Boogie code from Fortran language source(s)."""
```

Comment:
Please add more comments so that we know a year down the road what basically almost each line is doing. Especially comments hacks.